stix_shifter_modules/alertflex/configuration/alertflex_lang.json

```diff
@@ -0,0 +1,49 @@
+{
+    "connection": {
+        "host": {
+            "label": "Management IP address or Hostname",
+            "placeholder": "192.168.1.10",
+            "description": "Specify the IP address or hostname of the Alertflex controller"
+        },
+        "port": {
+            "label": "Host Port",
+            "description": "Set the port number that is associated with the Host name or IP"
+        },
+        "help": {
+            "label": "Help",
+            "description": "More details on the datasource setting can be found in the specified link"
+        },
+        "limit": {
+            "label": "Result Size Limit",
+            "description": "The maximum number of entries or objects that are returned by search query.The default result size limit is 1000. The value must not be less than 1 and must not be greater than 10,000."
+        },
+        "timeout": {
+            "label": "Query Timeout Limit",
+            "description": "The time limit in minutes for how long the query is run on the data source. The default time limit is 1. When the value is set to zero, there is no timeout. If the query takes longer than 1 min, it is likely to indicate a problem."
+        },
+        "concurrent": {
+            "label": "Concurrent Search Limit",
+            "description": "The number of simultaneous connections to the data source. The default limit for the number of connections is 4."
+        },
+        "cert": {
+            "label": "Synchronous dummy Connection Certificate",
+            "description": "Use self-signed SSL certificate and CA content(root and intermediate) of datasource"
+        },
+        "sni": {
+            "label": "Server Name Indicator",
+            "description": "The Server Name Indicator (SNI) enables a separate hostname to be provided for SSL authentication"
+        }
+    },
+    "configuration": {
+        "auth": {
+            "username": {
+                "label": "Username",
+                "description": "Username with access to the search API"
+            },
+            "password": {
+                "label": "Password",
+                "description": "Password of the user with access to the search API"
+            }
+        }
+    }
+}
```

Comment:
name this file `lang_en.json` instead. and made changes in the fields and structure as you do in config.json file